packaging/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "package_name": "openvstorage-health-check",
+    "product_name": "Open vStorage - Health-Check",
+    "source_contents": "--transform 's,^,{0}-{1}/,' config/healthcheck *.txt",
```

Comment:
I think you want to include more in your sources (the directories ```ovs```, ```config``` and probably others as well)